### DISCUSSION TITLE
4.6.3

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#157] [Announcements]
#### CHANGED 
- 启动应用时检查与刷新Cookie
  - ⚠引入外部服务器部署本项目的webapi进行时间戳加密，支持自己部署与配置
- 禁止方向键控制分集分p
#### FIXED
- 修复部分按钮图标位置没居中
- 调整CDN选择框布局，修复测速后展开显示不全
- 修复用户主页搜索视频403错误

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.3'>4.6.3</a>.</em>

**Comment by ADudu-L** [comment]
导出下载的视频总是失败
![image](https://github.com/ywmoyue/biliuwp-lite/assets/76568099/e82b0310-bc38-4687-963b-1b7b82a14139)

**Reply by ywmoyue** [reply]
可能我启用ci编译导致依赖出问题了，下个版本修复
